### ISSUE TITLE
Backspace doesn't work when entering URL

### ISSUE THREAD

**Issue by sumpygump** [a-h/min#3]
I am not sure what is going on but the backspace key does not work when I am entering a URL.

![image](https://user-images.githubusercontent.com/676007/94499822-456db400-01c3-11eb-9d59-9a77cd7f176e.png)

This is in MacOS Terminal.

**Comment by a-h** [comment]
Thanks for trying it out and leaving the report!

I use the MacOS terminal myself and it's working great for me so I'll try it out on another mac user and get back to you.

**Comment by a-h** [comment]
I found another Mac user that doesn't use the terminal and so, has left it completely vanilla and it didn't work for them (although it worked fine on BSD and Linux machines I tried it on).

It appears I've updated my zsh config to send ctrl-h, which is why it worked for me.

I think I've fixed this in https://github.com/a-h/min/commit/2ef6e8c256fafb21810bca9817f43b3a109a3182

That's out as release https://github.com/a-h/min/releases/tag/v0.0.39

I haven't tested it on the same machine yet, to see if that resolves it. If you have chance to test it out, that would be great. Thanks!
